### DISCUSSION TITLE
Ensuring the Distal Fibula Placement

### DISCUSSION THREAD

**Discussion by qferreira-1** [lassoan/SlicerBoneReconstructionPlanner#107] [General]
Hello All,

Looking for some feedback or possible solutions to a problem that I am running into. Slicer 5.2.2 is being used. I am doing a mandible, left fibula free flap and the distal fibula needs to be placed against the left ramus and the proximal segment needs to go to the right anterior mandible. For some reason, no matter how I place the planes or orientate them, it is always placing the proximal fibula against the left ramus. Is there a way to guarantee the placement of the distal vs proximal ends?

**Comment by qferreira-1** [comment]
![image](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/assets/122468415/d85711ad-a923-4193-a329-2a332ac20b6f)

Included a photo for reference. The blue plane should be the distal plane yet it is showing the proximal fibula segment against the left ramus. I have tried redrawing the fibula line, and mandible curve, placing new planes in various orientations. But for some reason, it is not displaying correctly. Let me know your thoughts.

**Comment by cmfsx** [comment]
it is the direction of the fibula line. Draw the fibula line the other way
around ( if you start the line at proximal end start drawing from the
distal end )

**Reply by qferreira-1** [reply]
Is there a rule of thumb for placement? Usually I place distal end first, and am able to orientate the first plane to be the distal segment and last plane to be proximal.
